README.md

```diff
@@ -1,5 +1,51 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Portfolio
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+> My Professional Portfolio.
+
+![screenshot](./app_screenshot.png)
+
+This website describes my Professional Experience, Achievements and Personal Projects.
+
+## Built With
+
+- HTML & CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] **\[OPTIONAL\]** It would be great to have a live preview link in the readme.